stable/insights-agent/values.yaml

```diff
@@ -23,6 +23,7 @@ cronjobs:
   successfulJobsHistoryLimit: 2
   nodeSelector: null
   runJobsImmediately: true
+  dnsPolicy:
```

Comment:
I think maybe we should comment this out? Not sure how helm would interpret this as-is